sample_kotlin/build.gradle

```diff
@@ -43,6 +43,8 @@ dependencies {
 
     implementation 'com.google.firebase:firebase-core:16.0.9'
     implementation 'com.google.firebase:firebase-messaging:18.0.0'
+
+    implementation 'com.squareup.moshi:moshi-kotlin:1.9.2'
```

Comment:
Why do we need to add moshi dependency to the sample_kotlin module?

Reply:
Yes, I'd expect things to work without this, so I think it's best if we don't include it here.

Reply:
I've removed this, it was useful to test that the demo app still worked correctly with this dependency (previously the app would crash).